content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+    ```shell
+    make push-git-tag-dev
+    ```
+1. ArgoCD syncs nginx ingress and cert-manager automatically
+1. Manually update the DNS records so that ArgoCD is accessible.
+    ```shell
+    # Apply below DNS CNAME record manually in AWS Route53 for below:
+    #   argodemo.lab.kubermatic.io
+    #   *.argodemo.lab.kubermatic.io
+    #   grafana-user.self.seed.argodemo.lab.kubermatic.io
+    #   alertmanager-user.self.seed.argodemo.lab.kubermatic.io
+    # You can get load balancer details from `k get svc -n nginx-ingress-controller nginx-ingress-controller`
+    # After DNS setup, you can access ArgoCD at https://argocd.argodemo.lab.kubermatic.io
+    ```
+1. Install KKP EE without helm-charts. If you would want complete ArgoCD setup with separate seeds, we will need Enterprise Edition of KKP. You can run the demo with master-seed combo. For this,  community edition of KKP is sufficient.
+    ```shell
+    make install-kkp-dev
+    ```
+1. Apply ClusterIssuer. **Note:** For the demo, staging LetsEncrypt certificate provider is used. In real world production applications, you should use production LetsEncrypt certificate issuer. Just need to change the url in `dev/clusterIssuer.yaml` file.
+    ```shell
+    kubectl apply -f dev/clusterIssuer.yaml
+    ```
+1. Add seed for self (need manual update of kubeconfig in seed.yaml)
+    ```shell
+    make create-long-lived-master-seed-kubeconfig
+    # commit changes to git and push latest changes in
+    make push-git-tag-dev
+    ```
+1. Sync all apps in ArgoCD by accessing ArgoCD UI and syncing apps manually
+1. Seed DNS record AFTER seed has been added (needed for usercluster creation). Seed is added as part of ArgoCD apps reconciliation above
+    ```shell
+    # Apply DNS record manually in AWS Route53
+    # *.self.seed.argodemo.lab.kubermatic.io
+    # Loadbalancer details from `k get svc -n kubermatic nodeport-proxy`
+    ```
+1. Access KKP dashboard at https://argodemo.lab.kubermatic.io
+1. Now you can create user-clusters on this master-seed cluster
+1. (only for staging letsencrypt) We need to provide the staging letsencrypt cert so that monitoring IAP components can work. For this, one needs to save the certificate issuer for `https://argodemo.lab.kubermatic.io/dex/` from browser / openssl and insert the certificate in `dev/common/custom-ca-bundle.yaml` for the secret `letsencrypt-staging-ca-cert` under key `ca.crt` in base64 encoded format. Post saving the file, commit the change to git and re-apply the tag via `make push-git-tag-dev` and sync the ArgoCD App .
+
+### Installation of dedicated KKP seed
+> **Note:** You can follow these steps only if you have a KKP EE license with you. With KKP CE licence, you can only work with one seed (which is master-seed combo above)
```

Comment:
No "with you", that sounds like someone is going for a walk and should not forget their phone or something.

Reply:
😆 corrected.